apps/sap-commerce-cloud/frontend/src/api/dataTransformers.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { productTransformer, baseSiteTransformer } from './dataTransformers';
+
+describe('dataTransformers', () => {
+  describe('productTransformer', () => {
+    it('transforms product data', () => {
+      const apiEndpoint = 'http://localhost:9002';
+      const skuIdsToSkusMap = { '123': 'product1' };
+      const baseSite = 'electronics-spa';
+      const item = {
+        id: '123',
```

Comment:
Just a small suggestion to use the existing product mock here: https://github.com/contentful/apps/blob/feat/sap-app-refactor-ext5483/apps/sap-commerce-cloud/frontend/src/__mocks__/mockProductPreview.ts and to see if there are any other mocks used in a lot of these tests that you can extract into the mocks directory?

Reply:
can do! thanks :-D

Reply:
honestly I think A LOT of stuff could be extracted